nix/t1/mlirbc.nix

```
@@ -43,13 +43,18 @@ stdenvNoCC.mkDerivation {
   '' + lib.optionalString (!use-binder) ''
     firtool elaborate/*.fir \
       --annotation-file elaborate/*.anno.json \
+      --emit-bytecode \
+      --parse-only \
+      -o $out/${target}-${configName}-debug.mlirbc
```

```suggestion
      -o $out/${target}-${configName}-parse-only.mlirbc
```

`debug` implies the compile mode, suggesting a new name.

---

OK use this.